optimism/client.go

```diff
@@ -565,7 +568,8 @@ func (ec *Client) erc20TokenOps(
 			continue
 		}
 
-		if _, ok := ec.supportedTokens[strings.ToLower(contractAddress)]; !ok {
+		_, ok = ec.supportedTokens[strings.ToLower(contractAddress)]
+		if ec.filterTokens && !ok {
```

Comment:
This changes the default behavior of whitelisting erc20 tokens. Just wanna point this point to make sure this is this desired.

Reply:
yeah this is desired. We want to bypass the whilelist in rosetta.